src/pwa-buildpack/project-setup/index.md

```diff
@@ -8,22 +8,34 @@ This multi-part tutorial goes through the steps of setting up a development envi
 
 1. [Create the initial theme files]
 1. [Install project dependencies]
+1. [Link project to the Magento backend]
 1. [Create configuration files]
 1. [Create a simple peregrine app]
 
 ## Prerequisites:
 
 To complete this tutorial, make sure your development system has the following:
 
-* [NPM]
-* [NodeJS 8.x]
-* A local Magento 2 store accessible via filesystem.  
+* [NPM] - NPM is a package manager for JavaScript.
```

Comment:
Do you think we should give context to the expected reader here and say "NPM is like Composer for JS--in fact, Composer was inspired by NPM!"